cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -195,6 +214,40 @@ To configure a custom SAML authentication method:
 1. Download metadata required by your IdP. Click **Download**. Open the file and make a note of the following values:<ul><li><b>Entity ID</b>: The <code>entityID</code> attribute of the <code>&lt;EntityDescriptor&gt;</code> tag.</li><li><b>Login URL</b>: The <code>location</code> attribute of the <code>&lt;AssertionConsumerService&gt;</code> tag.</li></ul>
 1. In the browser where you are logged in to your IdP, update the authentication configuration to use the Entity ID and Login URL from the metadata file.
 1. Configure the SAML assertions that your IdP sends to {{ site.data.products.db }}. Your IdP must send an assertion with a `name` field and a second assertion with an `email` field, each mapped to the relevant fields in your IdP. Refer to the documentation for your IdP.
+1. To configure SCIM provisioning, refer to [(Optional) Configure SCIM autoprovisioning on Okta](#configure-scim-provisioning).
+
+<a id="configure-scim-provisioning"></a>
+
+#### (Optional) Configure SCIM autoprovisioning on Okta
+
+The exact steps and requirements for enabling SCIM autoprovisioning depend upon your IdP. To add support for [SCIM autoprovisioning](#scim-autoprovisioning) on Okta:
+
+1. Log in to Okta Admin Dashboard as an admin user.
+1. Click **Applications** and edit the [SAML](#saml) application for your {{ site.data.products.db }} organization.
+1. Edit the SAML application, then click **Provisioning**:
```

Comment:
I think the Provisioning tab doesn't show up until you select SCIM under general settings.